lib/protocol/bep.proto

```diff
@@ -122,6 +122,7 @@ message FileInfo {
     bool deleted        = 6;
     bool invalid        = 7 [(gogoproto.customname) = "RawInvalid"];
     bool no_permissions = 8;
+    bool hidden         = 18;
```

Comment:
Why 18?

Reply:
I didn't notice that 14 and 15 were available. I can change that to either. I've read that field numbers below 16 take only one byte so it might actually be a good idea.

Reply:
The fields are now ordered to create a reasonable struct definition, which blurs the allocation order. But I think that for this reason we used lower numbered fields first, and had blocks + seldom used ones at 16+ for this reason. I think hidden works fine at 18 because it's almost never going to be set and even then just becomes two bytes.